File: grade.py
#Accept the percentage from the user and display the grade according to the following 
#criteria. 
a=int(input("enter the percentage: "))
if(a>=85 and a<=100):
    print("grade A+")
elif(a>=65 and a<=85):
    print("grade A")
elif(a>=45 and a<65):
    print("grade B")
elif(a>=25 and a<45):
    print("grade C")
else:
    print("grade D")

File: Grade.java

import java.util.Scanner;
public class Grade {
    public static void main(String[] args) {
        int n;
        Scanner sc= new Scanner(System.in);
        System.out.println("Enter the percentage of the student:");
        n=sc.nextInt();
        if(n>=95 &&n<=100){
            System.out.println("Grade A+");
        } else if(n>=90 &&n<95){
            System.out.println("Grade A");
        } else if(n>=85 && n<90){
            System.out.println("Grade B+");
        } else if(n>=80 && n<85){
            System.out.println("Grade B");
        } else{
            System.out.println("Grade C");
        }
        sc.close();

    }
    
}
